<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IcqMainSettings</class>
 <widget class="QWidget" name="IcqMainSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="avatarBox">
     <property name="text">
      <string>Don't send requests for avatarts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="reconnectBox">
     <property name="text">
      <string>Reconnect after disconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Codepage: &lt;/span&gt;(note that online messages use utf-8 in most cases)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="codepageBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>System</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Apple Roman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Big5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Big5-HKSCS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">EUC-JP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">EUC-KR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">GB18030-0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">IBM 850</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">IBM 866</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">IBM 874</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 2022-JP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ISO 8859-16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Bng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Dev</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Gjr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Knd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Mlm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Ori</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Pnj</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Tlg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Iscii-Tml</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">JIS X 0201</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">JIS X 0208</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">KOI8-R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">KOI8-U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">MuleLao-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ROMAN8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Shift-JIS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">TIS-620</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">TSCII</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">UTF-8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">UTF-16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">UTF-16BE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">UTF-16LE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1250</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1251</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1252</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1253</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1254</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1255</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1257</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1258</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">WINSAMI2</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>67</width>
       <height>187</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
